src/kubeEvents.ts

```diff
@@ -0,0 +1,25 @@
+import { Kubectl } from './kubectl';
+import * as kubectlUtils from './kubectlUtils';
+import * as explorer from './explorer';
+
+export enum EventDisplayMode {
+    Show,
+    Follow
+}
+
+export async function getEvents(kubectl: Kubectl, displayMode: EventDisplayMode, explorerNode?: explorer.ResourceNode) {
+    let eventsNS;
+
+    if (explorerNode) {
+        eventsNS = explorerNode.id;
+    } else {
+        eventsNS = await kubectlUtils.currentNamespace(kubectl);
+    }
+
+    let cmd = ` get events --namespace ${eventsNS}`;
+    if (EventDisplayMode.Follow === displayMode) {
+        cmd += ' -w';
+    }
+
+    kubectl.invokeInNewTerminal(cmd, 'Kubernetes Events');
```

Comment:
I'd suggest doing an `invokeInSharedTerminal` for Show (to avoid spawning a new terminal for every show) and `invokeInNewTerminal` for Follow (to avoid blocking the shared terminal).

Reply:
i get you now, sorry I should have handled that way.